torch_xla/utils/checkpoint.py

```diff
@@ -8,6 +8,18 @@
 from typing import Any, Iterable, List, Tuple, Union
 
 
+def extract_tensors_from_list(inputs):
```

Comment:
nit, would it make sense to move this inside the CheckpointFunction class? We don't have other use cases, yet -- and the function name is quite general.

Reply:
oh sure, will do.